### ISSUE TITLE
Add new text generation only endpoint for integrated models

### ISSUE THREAD

**Issue by biswaroop1547** [premAI-io/prem-services#41]
### Description

Would be nice to have a text generation only endpoint for testing out base models actually which is not basically instruction tuned or chat specific, if user wants to test or use just generation capabilities.

**Comment by filopedraz** [comment]
Is this still relevant @biswaroop1547? What do you think?

**Comment by biswaroop1547** [comment]
Yes @filopedraz as currently we don't have a "generation only" endpoint but only chat generation endpoint
